styles/components/blog.scss

```diff
@@ -0,0 +1,52 @@
+.container {
```

Comment:
I deleted this selector. it actually had no purpose other than to remind me how to use containers properly.